src/platform/web/ui/session/rightpanel/MemberDetailsView.js

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {AvatarView} from "../../AvatarView.js";
+import {TemplateView} from "../../general/TemplateView.js";
+
+export class MemberDetailsView extends TemplateView {
+    render(t, vm) {
+        const encryptionMessages = {
+            true: vm.i18n`Messages in this room are end-to-end encrypted.`,
+            false: vm.i18n`Messages in this room are not end-to-end encrypted.`
+        };
+        return t.div({className: "MemberDetailsView"},
+            [   t.view(new AvatarView(vm, 128)),
```

Comment:
Avatars loaded here are blurry; they're smaller than the requested size. Probably a synapse bug.

Reply:
So in Element Web, the avatar is sized to 2x30vh: https://github.com/matrix-org/matrix-react-sdk/blob/e78a1adb6f1af2ea425b0bae9034fb7344a4b2e8/src/components/views/right_panel/UserInfo.tsx#L1454-L1456 but the size is dynamic there depending on the right panel width. Prior to that it was 800x800, so if there is a synapse bug here, it might have been undetected so far. If you can accurately describe the problem you see with thumbnails, might be good to file an issue on https://github.com/matrix-org/synapse/issues/ and refer to that in the comment or commit for why 128 is not good enough. Also, make sure `window.devicePixelRatio` is 1 on the device/browser where you are seeing the problem, otherwise this is expected. On that subject, we should multiply the number here with `this._platform.devicePixelRatio` in any case.